admin-ui/app/routes/Apps/Gluu/GluuCommitDialog.tsx

```
@@ -86,264 +66,155 @@ const GluuCommitDialog = ({
-              }}
+
+  const modalContent =
+    (webhookModal || loadingWebhooks) && canReadWebhooks ? (
+      <>{webhookTriggerModal({ closeModal })}</>
+    ) : (
+      <>
+        <div
+          className={classes.overlay}
+          onClick={closeModal}
+          onKeyDown={handleOverlayKeyDown}
+          role="button"
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)